charts/hello-eks-anywhere/values.yaml

```diff
@@ -1,11 +1,6 @@
 namespace: eksa-packages
-
 sourceRegistry: "public.ecr.aws/aws-containers"
```

Comment:
Looks like this doubles up with repository on `aws-containers`

Reply:
Fixed, good point. The `aws-containers` is part of the registry name and the repository name is `hello-eks-anywhere` as far as ECR is concerned.